packages/design-system/src/themes/default/index.css

```diff
@@ -175,4 +175,10 @@
   --ads-emoji-reactions-count-text-color: var(--ads-old-color-dove-gray);
   --ads-emoji-reactions-count-active-text-color: var(--ads-old-color-rock-spray);
   --ads-emoji-reactions-count-hover-text-color: var(--ads-color-black-750);
+
```

Comment:
That will be little difficult to do for all these. Also, in that case we should sort our styles as well. And this will add more friction for developers. We can explore some plugin or linter config to automate this.

Reply:
I don't mean the individual variables, I mean the general grouping of variables